examples/pipeline/3_pipeline_dict_with_services_full.py

```diff
@@ -11,38 +11,38 @@
 from dff.core.engine.core import Context, Actor
 
 from dff.core.pipeline import CLIMessengerInterface, Service, Pipeline, ServiceRuntimeInfo
-from examples.pipeline._pipeline_utils import SCRIPT, get_auto_arg, auto_run_pipeline
+from _pipeline_utils import SCRIPT, should_auto_execute, auto_run_pipeline
```

Comment:
i think all functions can be migrate into `examples/utils`

Reply:
Again, all the imports will be reworked